packages/vite/CHANGELOG.md

```diff
@@ -1,20 +1,86 @@
 ## 4.0.0-beta.0 (2022-12-05)
 
-* chore: enable prettier trailing commas (#11167) ([134ce68](https://github.com/vitejs/vite/commit/134ce68)), closes [#11167](https://github.com/vitejs/vite/issues/11167)
+### Main Changes
+
+This major is smaller in scope compared to Vite 3, with the main objective of upgrading to Rollup 3. We've worked with the ecosystem to ensure a smooth upgrade path for this new major.
+
+#### Rollup 3
+
+Vite is now using [Rollup 3](https://github.com/vitejs/vite/issues/9870), which allowed us to simplify Vite's internal asset handling and has many improvements. See the [Rollup 3 release notes here](https://github.com/rollup/rollup/releases).
+
+#### Framework Plugins out of the Vite core monorepo
+
+[`@vitejs/plugin-vue`](https://github.com/vitejs/vite-plugin-vue) and [`@vitejs/plugin-react`](https://github.com/vitejs/vite-plugin-react) have been part of Vite core monorepo since the first versions of Vite. This helped us to get a close feedback loop when making changes as we were getting both Core and the plugins tested and released together. With [vite-ecosystem-ci](https://github.com/vitejs/vite-ecosystem-ci) we can get this feedback with these plugins developed on independent repositories, so from Vite 4, [they have been moved out of the Vite core monorepo](https://github.com/vitejs/vite/pull/11158). This is meaningful for Vite's framework-agnostic story, and will allow us to build independent teams to maintain each of the plugins. If you have bugs to report or features to request, please create issues on the new repositories moving forward: [`vitejs/vite-plugin-vue`](https://github.com/vitejs/vite-plugin-vue) and [`vitejs/vite-plugin-react`](https://github.com/vitejs/vite-plugin-react).
+
+#### New React plugin using SWC during development
+
+[SWC](https://swc.rs/) is now a mature option for [Babel](https://babeljs.io/). SWC's Fast Refresh implementation is a lot faster than Babel, and for some projects, it is now a better alternative.
```

Comment:
Is a word missing in the first sentence or this is an english form I'm not used to?

Reply:
Should it be:

>SWC is now a mature replacement for Babel.

?

Reply:
this sounds better for me, but maybe french bias